packages/Input/src/deprecated/Input.tsx

```diff
@@ -77,7 +77,7 @@ const Input: FunctionComponent<IInputProps> = ({
 
   return (
     <div className={`form-field form-field--input ${modifierClassName} ${passwordClassName}`}>
-      <label htmlFor={attrs.id} className={requiredClassName}>
+      <label htmlFor={attrs.id || attrs.name} className={requiredClassName}>
```

Comment:
I'm wondering if we are setting an automatic "id" do you think we should prefix it e.g. `input-${name}` in case the "id" clashes with another "id" on a page e.g. `<div id="email">`